integrations/ScriptLoader.js

```diff
@@ -4,6 +4,7 @@ import logger from "../utils/logUtil";
 const defaultAsyncState = true;
 
 export const LOAD_ORIGIN = "RS_JS_SDK";
+export const IS_NATIVE_SCRIPT = "true";
```

Comment:
There is no need to make this a constant.

Reply:
Addressed